Source/WebCore/html/HTMLScriptElement.h

```diff
@@ -36,9 +39,16 @@ class HTMLScriptElement final : public HTMLElement, public ScriptElement {
     static Ref<HTMLScriptElement> create(const QualifiedName&, Document&, bool wasInsertedByParser, bool alreadyStarted = false);
 
     String text() const { return scriptContent(); }
-    WEBCORE_EXPORT void setText(String&&);
+    WEBCORE_EXPORT ExceptionOr<void> setText(std::variant<RefPtr<TrustedScript>, String>);
+
+    String textContentForBindings() const;
+    ExceptionOr<void> setTextContentForBindings(std::optional<std::variant<RefPtr<TrustedScript>, String>>);
```

Comment:
Why doesn’t this take a `&&` or at least a `const&`? We may be able to follow an idiom with a bit less reference count churn.